R/predict_classprob.R

```diff
@@ -48,3 +48,16 @@ predict_classprob.model_fit <- function(object, new_data, ...) {
 # @inheritParams predict.model_fit
 predict_classprob <- function(object, ...)
   UseMethod("predict_classprob")
+
+check_spec_levels <- function(spec) {
+  if ("class" %in% spec$lvl) {
+    rlang::abort(
+      glue::glue(
+        "The outcome variable `{spec$preproc$y_var}` has a level called 'class'. ",
+        "This level is reserved for parsnip's classification internals; please ",
```

Comment:
```suggestion
        "This value is reserved for parsnip's classification internals; please ",
```
Is it confusing to say it is a `level` internally? It's not really, is it?

Reply:
It may be confusing, I agree. Your suggestion is better!